pkg/service/types.go

```diff
@@ -28,14 +27,13 @@ type MeshService struct {
 	// If the service resides on a Kubernetes service, this would be the Kubernetes namespace.
 	Namespace string
 
-	// The name of the service. May include instance (e.g. pod) information if the backing service
-	// doesn't have a single, stable ip address. For example, a MeshService created by a headless
-	// Kubernetes service named mysql-headless, will have the name "mysql.mysql-headless"
-	// A MeshService created by a normal ClusterIP service named mysql will be named "mysql"
-	// This imposes a restriction that service names cannot contain "." (which is already
-	// the case in kubernetes). Thus, MeshService.Name will be of the form: [subdomain.]providerKey
+	// The name of the service. This will not include a subdomain in the case that the service is backed by a single
+	// pod.
```

Comment:
This comment was a bit confusing, should this be something like clusterip service will not have subdomains?

Reply:
Could the name always exclude the subdomain regardless of whether or not the service is backed by a single pod?